### ISSUE TITLE
Debian build: Architecture is not recognised in Debian 11

### ISSUE THREAD

**Issue by adriangibanelbtactic** [ONLYOFFICE/document-server-package#234]
**Describe your problem**:

When building Debian package it fails because of not detecting proper architecture.

**Steps to reproduce the problem**:
- Build _documentserver_ thanks to _build_tools_ in /root/build_tools directory.

Try to package thanks to:
```
apt install build-essential m4 npm
npm install -g pkg

cd /root
git clone https://github.com/ONLYOFFICE/document-server-package.git
cd document-server-package/deb
apt build-dep ./
cd /root/document-server-package

PRODUCT_VERSION='7.0.0' BUILD_NUMBER='132' make deb
```

**What were you expecting to happen**
The package is built and no error happens.

**What happens instead**

Sorry, I don't have the full output.

Basically it was trying to read `../build_tools/out/linux_/onlyoffice/documentserver/` while it had to read `../build_tools/out/linux_64/onlyoffice/documentserver/` instead.

Not finding those files halted the Makefile process.

**Proposed fix**

Not sure.

The problem relies on the `uname -p` command. I guess it usually outputs amd64 or i386.
In Debian 11 it outputs `unknown`.

Maybe add an alternative way of knowing the current platform if `uname -p` command outputs `unknown`. Probably using: `uname -m` which outputs: `x86_64` for me.

**Workaround**

This is a quick-and-dirty workaround so that you can finally build the package in my machine:
`sed -i 's/TARGET := $(PLATFORM)_$(ARCHITECTURE)/TARGET := $(PLATFORM)_64/g' Makefile`
.

**document-server-package base repo version**: v7.0.0.132

**Operating system**: Debian 11 Netinst

**Compiler version**: N/A.

**Comment by ShockwaveNN** [comment]
Thanks for your report

I can confirm that output of `uname -p` differs on Ubuntu 20.04 and Debian 11:
```
lobashov@lobashov-3:~/temp$ docker run -it ubuntu:20.04 uname -p
x86_64
lobashov@lobashov-3:~/temp$ docker run -it debian:11 uname -p
unknown
```

I've created issue 55073 to correctly handle this situation

**Comment by ShockwaveNN** [comment]
Hi this issue should be fixed via https://github.com/ONLYOFFICE/document-server-package/pull/259

Please note that it's only on `develop` branch and this version may include any number of bugs
If everything will be well - this fix will be included in not next, but next-next major version, it may be v7.2

**Comment by ShockwaveNN** [comment]
The fix is part of v7.1.0 release so issue should be fixed for stable

I'm closing this issue
